tests/v2/validation/Jenkinsfile_vpn

```diff
@@ -13,124 +13,127 @@ node {
     def envFile = ".env"
     def rancherConfig = "rancher_env.config"
     def branch = "release/v2.6"
-    if ("${env.branch}" != "null" && "${env.branch}" != "") {
-      branch = "${env.branch}"
+    if ("${env.BRANCH}" != "null" && "${env.BRANCH}" != "") {
+      branch = "${env.BRANCH}"
     }
     def repo = scm.userRemoteConfigs
     if ("${env.REPO}" != "null" && "${env.REPO}" != "") {
       repo = [[url: "${env.REPO}"]]
     }
     def timeout = "60m"
-    if ("${TIMEOUT}" != "null" && "${TIMEOUT}" != "") {
-      repo = [[url: "${TIMEOUT}"]]
+    if ("${env.TIMEOUT}" != "null" && "${env.TIMEOUT}" != "") {
+      timeout = "${env.TIMEOUT}" 
     }
-    wrap([$class: 'AnsiColorBuildWrapper', 'colorMapName': 'XTerm', 'defaultFg': 2, 'defaultBg':1]) {
-      withFolderProperties {
-        paramsMap = []
-        params.each {
+  withFolderProperties {
+    paramsMap = []
+    params.each {
+      if (it.value && it.value.trim() != "") {
           paramsMap << "$it.key=$it.value"
-        }
```

Comment:
Hey Markus could explain the reasoning behind removing the ` wrap(... `

Reply:
When @slickwarren and I were testing, the Jenkins job was failing. In debugging it further, we saw it's because the `Jenkinsfile_vpn` had multiple nuances not seen in `Jenkinsfile`. When we copied/pasted the old `Jenkinsfile` and added the VPN parameters, then that caused the Jenkins job to work properly.